frontend/src/components/projects/ProjectsPage.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import {Redirect, Route, Switch} from "react-router-dom";
+
+import ProjectsListContent from "./ProjectsListContent";
+import ProjectEditContent from "./ProjectEditContent";
+import PageContainer from "../PageContainer";
+
+const ProjectsPage = () => <PageContainer>
```

Comment:
There is missing the route for individual projects (such as /projects/:id which would have the ProjectsDetailContent component for the detailed information about a specific project)